shared/src/exceptions/invalid-credentials-error/invalid-credentials-error.exception.ts

```diff
@@ -0,0 +1,10 @@
+import { CustomExceptionName, ExceptionMessage } from '~/common/enums/enums';
+
+class InvalidCredentials extends Error {
```

Comment:
this isn't a reusable error, since you will need it only on the backend side. so left it only in the backend